lib/rubocop/cop/lint/percent_string_array.rb

```diff
@@ -55,7 +55,7 @@ def contains_quotes_or_commas?(node)
 
         def autocorrect(node)
           lambda do |corrector|
-            node.values.each do |value|
+            node.each_child_node do |value|
               range = value.loc.expression
 
               match = range.source.match(TRAILING_QUOTE)
```

Comment:
This means we're losing valuable information, and we're back to just fiddling with `children` of anonymous nodes, which is part of the problem node extensions solve. 

 The `#values` method should probably be `#elements`. (I can't remember if it's used for polymorphism, but if we change it, we'll see which tests break.)

Also feel free to change the `node` parameter to `array_node`. 🙂